src/main.rs

```diff
@@ -122,6 +123,14 @@ pub struct CommonArgs {
     #[clap(long)]
     pub custom_alpn: Option<String>,
 
+    /// The relay URL to use as a home relay,
+    ///
+    /// Can be set to "disabled" to disable relay servers and "custom"
+    /// to configure custom servers. The default is the n0 quickest responding
+    /// relay if the flag is not set.
+    #[clap(long, default_value_t = RelayModeOption::Default)]
+    pub relay: RelayModeOption,
```

Comment:
This help text needs a little tweaking I think.

You don't set this option to the literal value of "custom", but rather you give your relay url